### DISCUSSION TITLE
Run hrldas on a daily scale

### DISCUSSION THREAD

**Discussion by baitao225** [NCAR/hrldas#112] [Q&A] [answered]
I hope to run hrldas daily, so I downloaded the daily ERA5-LAND data by using the tool Daily statistics calculated from ERA5 data, but only nc format data could be downloaded by this method. The problem occurs in the./create_forcing.exe namelist.input.ERA5 step (because this step defaults ERA5-Land to grib format data).
How can I run hrldas daily? Should I input daily forcing data into the model or hourly data? If I should input daily forcing data into the model, how should I make the forcing data and modify the create_forcing.F file?
I'd appreciate it if you'd like to answer.

**Comment by andrewsoong** [comment, marked as answer]
The hrldas can not run with daily forcing! Also, 6 hourly data is about the maximum time step you could use as you will really be degrading the diurnal cycle of energy forcing.

**Reply by baitao225** [reply]
Thanks very much for your answer.
